src/library/dlgtrackinfo.cpp

```diff
@@ -387,8 +387,10 @@ void DlgTrackInfo::saveTrack() {
         QTableWidgetItem* colorItem = cueTable->item(row, 3);
         QTableWidgetItem* labelItem = cueTable->item(row, 4);
 
-        if (!rowItem || !hotcueItem || !colorItem|| !labelItem)
+        VERIFY_OR_DEBUG_ASSERT(!rowItem || !hotcueItem || !colorItem || !labelItem) {
```

Comment:
I think you need to invert the logic. It is verified that all these values are valid.